src/app/components/cards/coursecard/coursecard.component.ts

```diff
@@ -9,9 +9,11 @@ import {
 } from 'sunbird-sdk';
 import { InteractSubtype, InteractType, Environment, PageId } from '../../../../services/telemetry-constants';
 import { CommonUtilService } from '../../../../services/common-util.service';
-import { EnrollmentDetailsPage } from '@app/app/enrolled-course-details-page/enrollment-details-page/enrollment-details-page';
 import { Router } from '@angular/router';
+import forEach from 'lodash/forEach';
```

Comment:
Don't use loadash foreach , use default for loop.

Reply:
Okay, will make that change